index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portofolio</title>
+</head>
+<body class="Roland">
+    <header class="nav-bar">
+
+        <nav class="nav-list">
+            
+            <a class="logo" href="#">My Logo</a>
+
+            <button class="menu-button">
+                <img class="mobile-menu" src="images/nav.png" alt=" navigation" />
+            </button>
+
+        </nav>
+
+
+    </header>
```

Comment:
**_[OPTIONAL]_** - Please consider using flexbox to layout the content of this section

### Requirements screenshot 👇 

![image](https://user-images.githubusercontent.com/78251927/211936254-094efab7-b074-4f99-9c72-c7497b12a1dd.png)